eth2near/contract_wrapper/src/near_network_enum.rs

```diff
@@ -0,0 +1,55 @@
+use std::error::Error;
+use std::fmt;
+use std::fmt::{Display, Formatter};
+use std::str::FromStr;
+use serde::Deserialize;
+
+#[derive(Debug, Clone, Deserialize)]
+pub enum NearNetwork {
+    Mainnet,
+    Testnet,
+}
+
+#[derive(Debug, Clone, Deserialize)]
+pub struct IncorrectNearNetwork;
+
+impl Display for IncorrectNearNetwork {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(
+            f,
+            "Unknown near network id. Possible near networks: 'Mainnet', 'Testnet'"
+        )
+    }
+}
+
+impl Error for IncorrectNearNetwork {}
+
+impl Display for NearNetwork {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        match self {
+            NearNetwork::Mainnet => write!(f, "mainnet"),
+            NearNetwork::Testnet => write!(f, "testnet"),
+        }
```

Comment:
No need to duplicate the names of networks as `as_str()` is already implemented.

```suggestion
        write!(f, "{}", self.as_str())
```

Reply:
fixed: https://github.com/aurora-is-near/rainbow-bridge/pull/825/commits/e4e2d1194f7d4e6811452999d30c08b4ec39e34e